reference/api/indexes.md

```diff
@@ -122,12 +155,18 @@ You can use the response's `taskUid` to [track the status of your request](/refe
 
 <RouteHighlighter method="PATCH" route="/indexes/{index_uid}"/>
 
-Update an [index's](/learn/core_concepts/indexes.md) [primary key](/learn/core_concepts/primary_key.md#primary-key).  The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+Update an index's [primary key](/learn/core_concepts/primary_key.md#primary-key).
 
 You can freely update the primary key of an index as long as it contains no documents.
 
 To change the primary key of an index that already contains documents, you must first delete all documents in that index. You may then change the primary key, and finally, index your dataset again.
 
+### Path parameters
+
+| Name              | Type   | Description                                                               |
+| :---------------- | :----- | :------------------------------------------------------------------------ |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes.md#index-uid) of the requested index |
+
 ::: note
 It is not possible to change an index's `uid`.
 :::
```

Comment:
Could you move this up into the main endpoint description, please?
And could you also delete the following line about asynchronous tasks? Doesn't seem necessary to mention it on this endpoint (unlike the create index endpoint, which is one of the first endpoints accessed by new users of Meilisearch).